### ISSUE TITLE
ArgoCD extension doesn;t support Nongroupkind resources (POD)

### ISSUE THREAD

**Issue by sarabala1979** [argoproj/argo-cd#9411]
Checklist:

* [ ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [ ] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

Trying to add an extension for the POD resource. ArgoCD UI is not loading extension in resource page. But It is working in `rollout` resource. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version**

```shell
Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by sarabala1979** [comment]
@dsmith36

**Comment by Ziphone** [comment]
Fixed by #10010, @sarabala1979?
